Bottom: 6277ae55349fe63894717a51ca838498bf6e04b7
Top:    e23a440da4abcbed323748cf8ee422a21cb65f7c
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-18 11:07:06 +0300

Refresh of sock-res-mem.patch

---

diff --git a/ib-sock-int.h b/ib-sock-int.h
index 9d4cd30..18092c2 100644
--- a/ib-sock-int.h
+++ b/ib-sock-int.h
@@ -7,8 +7,6 @@
 #include <rdma/ib_verbs.h>
 #include <rdma/rdma_cm.h>
 
-#define IB_CQ_EVENTS_MAX 128
-
 #define IB_ADDR_TIMEOUT 100
 #define IB_ROUTE_TIMEOUT 100
